css/main.css

```diff
@@ -325,3 +326,54 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+/* Contact form style rules */
+.contact-form {
+  display: flex;
+  padding: 120px;
+}
+
+.form {
+  width: 50%;
+  padding: 0 20px;
+}
+
+.form-description {
+  color: #222;
+  font-size: xx-large;
+  font-weight: 900;
+  line-height: 40px;
+}
+
+input[type="text"],
+input[type="email"],
+textarea {
+  display: inline-block;
+  width: 48%;
+  border: none;
+  outline: none;
+  padding: 16px 12px;
+  border-bottom: 1px solid rgb(218, 218, 218);
+  margin: 4px 2px;
+  margin-bottom: 20px;
+}
+
+textarea {
+  width: 100%;
+  background-color: #fff7f2;
+  border-bottom: none;
+}
+
+input[type="email"] {
+  width: 100%;
+}
+
+.form-button {
```

Comment:
- [ ] Excellent work on the button color which Is an indicator of following the design. :smiley:  .
however, the button color changes depending on user interaction like`Hover, disabled ...`, in your case kindly at least add the `hover` effect of the form button. :cat:  :smile:  :100:

Reply:
Done.